tests/e2e/frontend-migration.spec.ts

```
@@ -0,0 +1,274 @@
+  return {
+    email: `test-${randomUUID().slice(0, 8)}@test.com`,
+    password: randomUUID(),
+  };
+}
+
+let keycloak: KeycloakTestHelper;
+let database: DatabaseTestHelper;
+
+// Extend Playwright test with testUser fixture
+const test = base.extend<{ testUser: TestUser }>({
+  testUser: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)